src/test/resources/logback-test.xml

```diff
@@ -1,7 +1,7 @@
 <configuration>
     <statusListener class="ch.qos.logback.core.status.NopStatusListener" />
 
-    <appender name="appender" class="me.karakelley.http.Appender">
+    <appender name="appender" class="me.karakelley.http.utility.Appender">
```

Comment:
I would rename this class to describe what it does. `InMemoryAppender`, `FakeAppender`, `MockAppender`, etc.

Reply:
(It's [not really a mock](https://martinfowler.com/bliki/TestDouble.html), though)